hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/TransactionInfoRepair.java

```diff
@@ -122,8 +122,7 @@ protected Component serviceToBeOffline() {
     }
   }
 
-  private DBColumnFamilyDefinition<String, TransactionInfo> getColumnFamily() {
-    Component component = serviceToBeOffline();
+  public DBColumnFamilyDefinition<String, TransactionInfo> getColumnFamily(Component component) {
```

Comment:
This method can be `static`, then we don't need to instantiate in the test.

```suggestion
  public static DBColumnFamilyDefinition<String, TransactionInfo> getColumnFamily(Component component) {
```